Formula/c/cucumber-cpp.rb

```diff
@@ -52,6 +55,9 @@ def install
       host: localhost
       port: 3902
     EOS
+    (testpath/"features/support/wire.rb").write <<~EOS
+      require 'cucumber/wire'
+    EOS
```

Comment:
Newer `cucumber` gem versions split out wire functionality into a separate gem which must be loaded separately.